Library/Formula/sfnt2woff.rb

```diff
@@ -0,0 +1,18 @@
+class Sfnt2woff < Formula
+  desc "Convert TTF and OTF fonts to WOFF format."
+  homepage "https://people.mozilla.org/~jkew/woff/"
+  url "https://people.mozilla.org/~jkew/woff/woff-code-latest.zip"
```

Comment:
Got it. Any guidance on how best to proceed when there isn't an official release other than latest? I'm happy to create a repo on GitHub for it, but that seems a bit artificial and not in spirit with the rule.